.pipelines/templates/ToolchainBuild.yml

```diff
@@ -27,7 +32,16 @@ steps:
     parameters:
       buildRepoRoot: ${{ parameters.buildRepoRoot }}
 
-  - bash: sudo make -C "${{ parameters.buildRepoRoot }}/toolkit" "-j$(nproc)" toolchain QUICK_REBUILD=y
+  - bash: |
+      additional_args=""
```

Comment:
Nit-pick: could we re-name `additional_args` to something more specific? For instance `daily_build_args`. I don't want to encourage bloating this variable with any future args, we may want to add here.

Reply:
Renamed it to be explicit and also split them up